### PR TITLE
Added results filename to write to if user does not specify path

### PR DESCRIPTION
**Pull request by terrameijar**
@kimberlythegeek I noticed a mistake that I made when coding this. `name` should point to a user specified path, If it is `None` then we need to create `results.json`. I had not added this in the last commit.

### CODE REVIEW

**Comment by terrameijar** [comment]
The latest commit adds the test requested in https://github.com/mozilla-services/axe-selenium-python/issues/159 . I know this should have been a separate PR but I had already pushed to this branch and the changes showed up.

File: axe_selenium_python/axe.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def write_results(self, data, name=None):
         if name:
             filepath = os.path.abspath(name)
         else:
-            filepath = os.path.join(os.path.getcwd(), name)
+            filepath = os.path.join(os.path.getcwd(), "results.json")
```
Comment (m8ttyB) [inline axe_selenium_python/axe.py]: Nit, update the method's docstring to note that `results.json` will be created in the current working directory if a filename and path aren't provided

Comment (terrameijar) [reply axe_selenium_python/axe.py]: Done

**Comment by terrameijar** [comment]
@kimberlythegeek flake8 error resolved.

**Comment by m8ttyB** [comment]
Merging this goodness, ty @terrameijar

**Pull request merged**